src/util/handlers/StaticThrowHandler.ts

```diff
@@ -0,0 +1,18 @@
+import type { HttpError } from '../errors/HttpError';
+import { AsyncHandler } from './AsyncHandler';
+
+/**
+ * Utility handler that can handle all input and always throws the given error.
+ */
+export class StaticThrowHandler extends AsyncHandler<any, never> {
+  private readonly error: HttpError;
```

Comment:
I suppose it could just be

```suggestion
  private readonly error: Error;
```

Reply:
It could, but since we always use HttpErrors internally (and want all classes to do this) I don't see any reason to make this more generic.